src/components/Unknown/BuildScreen/Filter/filters.ts

```diff
@@ -0,0 +1,67 @@
+import { Builder } from '../../../../../types/index';
+
+export type FilterOption = {
+  key: string;
+  value: string;
+};
+
+export type Filter = {
+  title: string;
+  key: string;
+  options: FilterOption[];
+};
+
+export type Filters = {
+  [key in Builder]: { accordion?: Filter[]; switcher?: Filter[] };
+};
+
+export const filters: Filters = {
```

Comment:
filters isn`t an array, filterByCategory for example (this is not the best name)